src/main/resources/skills.graphqls

```diff
@@ -3,19 +3,23 @@ schema {
 }
 
 type Query {
-  allSkills: [Skill!]!
   skill(id: String): Skill
-  allCategories: [Category!]!
+  skills: [Skill!]!
+  category(id: String): Category
+  categories: [Category!]!
+  skillsByCategory(categoryId: String): [Skill!]!
 }
 
 type Skill {
   id: ID!
   title: String!
   description: String!
+  examples: [String]
```

Comment:
How come this isn't `[String!]!` like the other lists? Would we ever return `null` for this field, or a list with `null` objects in? (rather than e.g. an empty list)

Reply:
let's discuss during our dev session

Reply:
agreed to populate db with empty list examples